### PR TITLE
Add blog post about CFP for OSLC 2017

### PR DESCRIPTION
**Pull request by caniszczyk**
http://events.linuxfoundation.org/events/open-source-leadership-summit

Signed-off-by: Chris Aniszczyk <caniszczyk@gmail.com>

### CODE REVIEW

**Comment by caniszczyk** [comment]
RFC, if I don't hear anything I will post this tomorrow :)

File: _posts/2016-11-29-oslc-tahoe.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+---
+
+The TODO Group will be hosting a formal open source office track at the Open Source Leadership Summit in Tahoe next year:
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">The new Open Source Leadership Summit, February in Lake Tahoe, is seeking speaking proposals thru Dec 14 at <a href="https://t.co/TJDWKF0r1U">https://t.co/TJDWKF0r1U</a> <a href="https://twitter.com/hashtag/lfosls?src=hash">#lfosls</a> <a href="https://t.co/efc2XIX0N1">pic.twitter.com/efc2XIX0N1</a></p>&mdash; LF Events (@EventsLF) <a href="https://twitter.com/EventsLF/status/797177088071569408">November 11, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+We hosted a track last year and encourage the wider community to <a href="http://events.linuxfoundation.org/events/open-source-leadership-summit/program/cfp">submit your proposals</a>. We are in pparticular looking for proposals in how companies have established open source programs (including their structure) and lessons learned in the process. We also love to hear about case studies in the process of open sourcing projects, from the good, the bad and the ugly :)
```
Comment (DuaneOBrien) [inline _posts/2016-11-29-oslc-tahoe.md]: Misspelled "particular"

**Comment by caniszczyk** [comment]
Thanks @DuaneOBrien!

File: _posts/2016-11-29-oslc-tahoe.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+---
+
+The TODO Group will be hosting a formal open source office track at the Open Source Leadership Summit in Tahoe next year:
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">The new Open Source Leadership Summit, February in Lake Tahoe, is seeking speaking proposals thru Dec 14 at <a href="https://t.co/TJDWKF0r1U">https://t.co/TJDWKF0r1U</a> <a href="https://twitter.com/hashtag/lfosls?src=hash">#lfosls</a> <a href="https://t.co/efc2XIX0N1">pic.twitter.com/efc2XIX0N1</a></p>&mdash; LF Events (@EventsLF) <a href="https://twitter.com/EventsLF/status/797177088071569408">November 11, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+We hosted a track last year and encourage the wider community to <a href="http://events.linuxfoundation.org/events/open-source-leadership-summit/program/cfp">submit your proposals</a>. We are in particular looking for proposals in how companies have established open source programs (including their structure) and lessons learned in the process. We also love to hear about case studies in the process of open sourcing projects, from the good, the bad and the ugly :)
```
Comment (DuaneOBrien) [inline _posts/2016-11-29-oslc-tahoe.md]: Taking a swing at the last two sentences: 

"We are especially interested in proposals that examine the structure of existing open source programs, what has helped those programs succeed (or fail!), how company culture has shaped those programs, and any lessons learned along the way. We would also love to present case studies that examine the process of open sourcing a project - the good, the bad and the ugly."

**Pull request merged**